ci/docker-compose.yml

```diff
@@ -55,7 +55,8 @@ services:
     command: server
     volumes:
       - authn-local:/run/authn-local
-      - ./authn-oidc/keycloak:/authn-oidc/keycloak/scripts
+      # TODO: authenticators_oidc/test has a dep on this
```

Comment:
is this a TODO that will be addressed in this PR?

Reply:
I will go thru and pull them out for remaining work.